examples/chart/teleport-cluster/templates/config.yaml

```diff
@@ -59,16 +59,18 @@ data:
     {{- if .Values.authenticationSecondFactor.secondFactor }}
         second_factor: {{ .Values.authenticationSecondFactor.secondFactor }}
     {{- end }}
-    {{- if .Values.authenticationSecondFactor.webauthn }}
+    {{- if not (or (eq .Values.authenticationSecondFactor.secondFactor "off") (eq .Values.authenticationSecondFactor.secondFactor "otp")) }}
```

Comment:
Drop this logic and always write it? It won't hurt.

Reply:
I considered this, but I think it'll just confuse people if they see that section appearing in their configs when second factor is disabled. The logic is a little more brittle but hopefully produces a better end result.